src/handbook/company/security/information-security.md

```diff
@@ -0,0 +1,195 @@
+---
+navTitle: Information Security Policy and Acceptable Use Policy
+---
+
+# Information Security Policy and Acceptable Use Policy
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @ZJvandeWeg    | 2023-05-01     |
+
+
+## Overview
+
+This Information Security Policy is intended to protect FlowForge’s employees,
+partners and the company from illegal or damaging actions by individuals, either
+knowingly or unknowingly.
+
+Internet-related systems, including but not limited to computer equipment,
```

Comment:
Does this imply our online assets? Because it might imply all laptops people work from are ours, which is not the case? In the scope section you break it out again, claiming people can work from their own devices?

Reply:
Agreed - will remove. I think the scope section is the more important one to define what the policy applies to. The language comes from Vanta template and in some places is more aligned to office-based working.